webservices/common/models/filings.py

```diff
@@ -123,8 +124,15 @@ def document_description(self):
         )
 
     @property
+    def amended_by(self):
+        if not self.superceded and not self.most_recent and self.amendment and len(self.amendment.longest_chain) > 0:
```

Comment:
Agreed, after looking at this after some sleep it looks like this could be simplified to one conditional.  Commit forthcoming.